src/node_options.cc

```diff
@@ -604,10 +604,14 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
   AddAlias("-pe", { "--print", "--eval" });
   AddAlias("-p", "--print");
   AddOption("--require",
-            "module to preload (option can be repeated)",
-            &EnvironmentOptions::preload_modules,
+            "cjs module to preload (option can be repeated)",
```

Comment:
I don’t think “cjs” or “CJS” are well-known enough shorthands.

```suggestion
            "CommonJS module to preload (option can be repeated)",
```

And elsewhere “cjs”/”CJS” is used.

Reply:
I think they’re quite well known, b it separately,  doesn’t CommonJS actually describe the thing that node mimicked, but node’s implementation is not actually compliant with that thing’s spec?